src/lib.rs

```diff
@@ -111,6 +111,9 @@ async fn run(app: AndroidApp) {
                 match event {
                     PollEvent::Main(event) => match event {
                         MainEvent::Destroy => {
+                            let player = &playerbox.as_ref().unwrap().player;
```

Comment:
I think I'd like the `if let Some(player) = playerbox.as_ref() {` here too, as it is in the other cases.